Gemfile.lock

```diff
@@ -45,7 +45,6 @@ GEM
       rake (>= 10.4, < 14.0)
     ast (2.4.0)
     aws-eventstream (1.0.3)
-
```

Comment:
It looks like this change was included by mistake. The gems didn't change so the whitespace in the lock file should not either.